.github/workflows/scripts/run-tests-logspout.sh

```diff
@@ -23,7 +23,7 @@ poll_ready "$cid_ls" "http://${ip_ls}/health"
 #   2021/01/07 16:14:52 # logspout v3.2.13-custom by gliderlabs
 #
 # which we expect to find by querying:
-#   docker.image:"docker-elk_logspout" AND message:"logspout gliderlabs"~3
+#   docker.name:"/docker-elk_logspout_1" AND message:"logspout gliderlabs"~3
```

Comment:
Note: This change may seem unrelated, but it was necessary because the value of this field happens to be a hash with Compose v2. I'll confirm this is still the case once v2 is officially released.

Reply:
Related to https://github.com/docker/compose-cli/issues/1972